src/stratis/errors.rs

```diff
@@ -66,7 +99,15 @@ impl fmt::Display for StratisError {
 impl Error for StratisError {
     fn cause(&self) -> Option<&dyn Error> {
         match *self {
-            StratisError::Error(_) | StratisError::Engine(_, _) => None,
+            StratisError::Msg(_) => None,
+            StratisError::Chained(_, ref chained) => Some(chained),
```

Comment:
Yes, I chose that explicitly but am open to discussing it. My main thought here is that we have the option of traversing all the way to the bottom of the error chain which I considered, but that removes all of the context of the error generated while chaining it when returning it. This seemed like it might not be what we want. On the other hand, it may also be useful to provide the initial causal error for all of this. I'm not set on this decision so I'd like to hear your feedback.

Reply:
Carry on and we'll see how it turns out.

Reply:
As far as I can tell, we don't call the `.cause()` method so let's discuss this at stand up because we have the flexibility now to define it however we want.

Reply:
Resolved by removing the definition.